spyglass/src/main/java/com/linkedin/android/spyglass/mentions/MentionsEditable.java

```diff
@@ -138,7 +138,9 @@ public MentionsEditable trim() {
         int len = length();
         while (len > 0 && Character.isWhitespace(charAt(len - 1))) {
             delete(len - 1, len);
-            len--;
+            //Note: we must reset len this way after delete to avoid IndexOutOfBoundsException crashes similar to what we saw
+            //happening with certain keyboards and {@link MentionsEditable#replace(int, int, CharSequence, int, int)}.
```

Comment:
Let's put a space after the "//" to stay consistent with how other comments are formatted in this file.
```suggestion
            // Note: we must reset len this way after delete to avoid IndexOutOfBoundsException crashes similar to what we saw
            // happening with certain keyboards and {@link MentionsEditable#replace(int, int, CharSequence, int, int)}.
```